mmdet3d/datasets/convert_utils.py

```diff
@@ -198,19 +198,19 @@ def get_kitti_style_2d_boxes(info: dict,
 
     Args:
         info (dict): Information of the given sample data.
-        cam_idx (int): Camera id which the 2d / mono3d annotations to obtain
-            belong to. In KITTI, typically only CAM 2 will be used,
+        cam_idx (int, optional): Camera id which the 2d / mono3d annotations to
```

Comment:
No need for "optional" in the latest version. We will only consider the input arguments "optional" when its default value is None.